reference/src/unit-testing.md

```diff
@@ -202,17 +201,16 @@ module pkg_addr::my_module {
     #[test]
     #[expected_failure(vector_error, minor_status = 0, location = Self)]
     fun vector_abort_wrong_minor_code() {
-        vector::borrow(&vector::empty<u64>(), 1);
+        vector::borrow(&vector<u64>[], 1);
     }
 }
 ```
 
 ### 5. `#[expected_failure]`
 
-This will pass if the test aborts with any error code. Because of this you
-should be incredibly careful using this way of annotating expected tests
-failures, and instead prefer one of the ways described above instead. Examples
-of these types of annotations are:
+This will pass if the test aborts with any error code. Because of this you should be incredibly
+careful using this way of annotating expected tests failures, and instead prefer one of the ways
+described above instead. Examples of these types of annotations are:
```

Comment:
```suggestion
This will pass if the test aborts with _any_ error code. You should be incredibly
careful using this to annotate expected tests failures, and always prefer one of the ways
described above instead. Examples of these types of annotations are:
```